src/blas/backends/cublas/cublas_batch.cpp

```diff
@@ -167,12 +167,25 @@ inline void gemm_batch_impl(sycl::queue &queue, transpose transa, transpose tran
             auto b_ = sc.get_mem<cuTypeB *>(b_acc);
             auto c_ = sc.get_mem<cuTypeC *>(c_acc);
             cublasStatus_t err;
-            CUBLAS_ERROR_FUNC_T_SYNC(
-                "cublasGemmStridedBatchedEx", cublasGemmStridedBatchedEx, err, handle,
-                get_cublas_operation(transa), get_cublas_operation(transb), m, n, k, &alpha, a_,
-                get_cublas_datatype<cuTypeA>(), lda, stride_a, b_, get_cublas_datatype<cuTypeB>(),
-                ldb, stride_b, &beta, c_, get_cublas_datatype<cuTypeC>(), ldc, stride_c, batch_size,
-                get_cublas_datatype<cuTypeS>(), cublas_gemm_algo);
+#ifdef SYCL_EXT_ONEAPI_ENQUEUE_NATIVE_COMMAND
+            CUBLAS_ERROR_FUNC_T("cublasGemmStridedBatchedEx", cublasGemmStridedBatchedEx,
+                                err, handle, get_cublas_operation(transa),
+                                get_cublas_operation(transb), m, n, k, &alpha, a_,
+                                get_cublas_datatype<cuTypeA>(), lda, stride_a, b_,
+                                get_cublas_datatype<cuTypeB>(), ldb, stride_b, &beta, c_,
+                                get_cublas_datatype<cuTypeC>(), ldc, stride_c, batch_size,
+                                get_cublas_datatype<cuTypeS>(), cublas_gemm_algo);
+#else
+            CUBLAS_ERROR_FUNC_T_SYNC("cublasGemmStridedBatchedEx",
```

Comment:
Why is this formatted differently from the thing it's replacing?

Reply:
There is a local clang format [here](https://github.com/oneapi-src/oneMKL/blob/develop/_clang-format). You should make sure you are using this config.

Reply:
I am delighted to see that we now have CI on oneMKL tests. Would another clang format job in CI be a good idea @Rbiessy ?

Reply:
We've not done anything on the GitHub CI but it is nice indeed. A clang format job is a good idea. I think we would need to discuss which clang-format version to use. We have seen differences between the internal CI which AFAIK uses clang-format-9. I think it would be easier to use the one shipped with DPC++ instead. I'll make a note to create an issue about that.

Reply:
For reference, created this issue https://github.com/oneapi-src/oneMKL/issues/580

Reply:
I don't know if there is an issue with clang-format configs.
I would suggest to download clang-format-9 and run `git-clang-format --binary /path/to/clang-format-9_linux-amd64 HEAD~8` to format all your changes.

Reply:
Fixed the format now, thanks.